src/components/Modal/FileErrorModal/FileErrorModal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import Modal from '..';
+
+interface FileErrorModalProps {
+  files: string[];
+  onClose: () => void;
+}
+
+const ErrorModal: React.FC<FileErrorModalProps> = ({
+  onClose,
+  files
+}: FileErrorModalProps) => {
+  return (
+    <Modal onClose={onClose}>
+      <ErrorHeader>There was an error importing some files.</ErrorHeader>
+      <ErrorText>Files that could not be imported:</ErrorText>
```

Comment:
We should add a `header` and `description` value to `FileErrorModalProps` to make it easier to reuse in the future.